src/data/lovelace/config/view.ts

```diff
@@ -27,7 +27,7 @@ export interface LovelaceBaseViewConfig {
 
 export interface LovelaceViewConfig extends LovelaceBaseViewConfig {
   type?: string;
-  badges?: Array<string | LovelaceBadgeConfig>;
+  badges?: LovelaceBadgeConfig[];
```

Comment:
we should still support just strings, no?

Reply:
Yes for backward compatibility, I reverted that change.